README.md

```diff
@@ -48,147 +26,49 @@ You can find the deployed project at [Tieme Ndo](https://tieme-ndo.netlify.com/)
 
 In the bid to transform smallholder farmers into semi-commercial farmers and agripreneurs, Tieme Ndo empower farmers to take advantage of business opportunities along the agricultural value chain through SAD (Small Agribusiness Development). The aim of SAD is to train farmers on entrepreneurship and value addition.
 
-### 4️⃣ Key Features
-
-- feature one
-- feature two
-- feature three
-- feature four
-- feature five
-
-## Tech Stack
-
-### Front end built using:
-
-#### React.js (Create-React-App)
-
-🚫 Why did you choose this framework?
-
-- Zero-setup, plug-and-play functionality.
-- Rich ecosystem of applications that will help in multiple aspects of the development.
-- Team-wide familiarity & experience
-
-#### Styled-Components
-
-🚫 Why did you choose this framework?
-
-- Component-specific styling & decentralization of styling
-- Team-wide familiarity & experience
-
-#### Front end deployed to `Netlify`
-
-# APIs
-
-## 2️⃣ Authentication API here
-
-🚫Replace text below with a description of the API
-
-Water's like me. It's laaazy ... Boy, it always looks for the easiest way to do things A little happy sunlight shining through there. Let all these little things happen. Don't fight them. Learn to use them. Even the worst thing we can do here is good.
-
-# 3️⃣ Environment Variables
-
-In order for the app to function correctly, the user need not set up their own environment variables at this point in time.
-
-Create an account on [Cloudinary](https://cloudinary.com/) to get preset value and cloudinary url.
-
-Create a `.env` file on your root directory and the following lines
-
-`REACT_APP_CLOUDINARY_PRESET=<Preset value gotten from cloudinary>`
-`REACT_APP_CLOUDINARY_URL=<API Base URL gotten from cloudinary>`
```

Comment:
Everything above that is deleted must stay in the `README`. 
The only exception are lines that start with this emoji: 🚫  ...those lines should be deleted.